enhancedDate.js

```diff
@@ -0,0 +1,85 @@
+/* Scott Henderson
+ * JSCRIPT 300 A Spring 2016
+ */
+
+/* exported enhancedDate */
+
+// IIFE === Immediately Invoked Function Expression
+var enhancedDate = (function iife() {
+	var _date;

```

Comment:
Leaving it undefined works, but best practice would be to initialize to null and check against null.